share/pot/common.sh

```diff
@@ -41,9 +41,21 @@ _POT_DEFAULT_raw_sockets_D='NO'
 _POT_DEFAULT_sysvipc_N='allow.sysvipc'
 _POT_DEFAULT_sysvipc_T='bool'
 _POT_DEFAULT_sysvipc_D='NO'
+_POT_DEFAULT_sysvshm_N='sysvshm'
```

Comment:
Hm, doesn't this interfere with setting `sysvipc` to `YES`?

From the man page:

```
             allow.sysvipc
                     A process within the jail has access to System V IPC
                     primitives.  This is deprecated in favor of the per-
                     module parameters (see below).  When this parameter is
                     set, it is equivalent to setting sysvmsg, sysvsem, and
                     sysvshm all to “inherit”.
```

Reply:
If someone were to enable both at once I suppose it would, but I left it in because I thought it's best to leave the options to the user. Of course if you'd prefer to have allow.sysvipc removed I'd change that. Should I do that?

Reply:
@zilti But I don't think this is what your logic does. By setting defaults (e.g., `_POT_DEFAULT_sysvshm_D='disable'`), these always override `allow.sysvipc` when creating a new pot. In other words (without having tested this myself), setting `allow.sysvipc` will basically have no effect whatsoever in this case, since it is always overridden by the new defaults (which means, that existing setups will break).

Reply:
I see. Would it work to just set empty defaults? E.g. `_POT_DEFAULT_sysvshm_D=''` ?

Reply:
@zilti Sorry for not following up on this, very busy. I'll try to look into this soon.